### ISSUE TITLE
PHP v8 support

### ISSUE THREAD

**Issue by sebastiansulinski** [gentlero/bitbucket-api#15]
Can you bump `kriswallsmith/buzz` dependency as I cannot seem to be able to use your package with v8 - getting dependency error:

```
Problem 1
    - Installation request for kriswallsmith/buzz v0.16.1 -> satisfiable by kriswallsmith/buzz[v0.16.1].
    - kriswallsmith/buzz v0.16.1 requires php ^5.4 || ^7.0 -> your PHP version (8.0.0) does not satisfy that requirement.
  Problem 2
    - kriswallsmith/buzz v0.16.1 requires php ^5.4 || ^7.0 -> your PHP version (8.0.0) does not satisfy that requirement.
    - gentle/bitbucket-api 1.1.2 requires kriswallsmith/buzz ^0.16 -> satisfiable by kriswallsmith/buzz[v0.16.1].
    - Installation request for gentle/bitbucket-api 1.1.2 -> satisfiable by gentle/bitbucket-api[1.1.2].
```

**Comment by GrahamCampbell** [comment]
https://github.com/BitbucketPHP/Client supports PHP 8, all of the API, and is not coupled to any HTTP client.

**Comment by sebastiansulinski** [comment]
Is this package no longer maintained?

**Comment by GrahamCampbell** [comment]
It doesn't look like it is. Bitbucket has made major changes to their endpoints, to use workspaces instead of users and teams, but this package still uses the old, deprecated endpoints.

**Comment by vimishor** [comment]
@sebastiansulinski First let me start by apologizing to you and the rest of the users of this package, because you lost time with this unmaintained package. I completely forgot to archive it and because of that any user could assume that the development didn't stopped. 

The long story is that I migrated to self hosting a while back and although the desire to continue maintaining this packages was mostly still there, I lacked at free time department and I also went through a burnout recently ( _better but not completely gone_ ) which didn't helped.

If the package had more contributors maybe it could continue to live during this time, but meh, it is what it is.

That being said, I encourage everyone to use another package instead of this one. It looks like @GrahamCampbell did a good job with [BitbucketPHP/Client](https://github.com/BitbucketPHP/Client) and probably there are others. I will archive this project on Packagist ASAP and by the end of the year I will also archive it here, on Github.
